README.md

```diff
@@ -1,6 +1,6 @@
 # Cytrus-RE
 
-[![Open Source Helpers](https://www.codetriage.com/rexogamer/cytrus-re/badges/users.svg)](https://www.codetriage.com/rexogamer/cytrus-re) [![Discord ](https://img.shields.io/discord/739524957255630858?color=Bonk&label=discord&logo=asd&logoColor=ad)](https://discord.gg/BTKT53N) [![Webpage](https://img.shields.io/website?down_color=lightgrey&down_message=is%20down.&up_message=is%20up%21&url=https%3A%2F%2Fcytrus-re.github.io%2F)](https://cytrus-re.github.io) [![Wiki](https://img.shields.io/badge/Wiki-information%20about%20cytrus--re-informational)](https://github.com/Cytrus-RE/cytrus-re/wiki) ![Size](https://img.shields.io/github/repo-size/Cytrus-RE/cytrus-re?label=Cytrus-RE%20Size) [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/cytrus-re/cytrus-re)
+[![Open Source Helpers](https://www.codetriage.com/rexogamer/cytrus-re/badges/users.svg)](https://www.codetriage.com/rexogamer/cytrus-re) [![Discord ](https://img.shields.io/discord/739524957255630858?color=Bonk&label=discord&logo=asd&logoColor=ad)](https://discord.gg/BTKT53N) [![Webpage](https://img.shields.io/website?down_color=lightgrey&down_message=is%20down.&up_message=is%20up%21&url=https%3A%2F%2Fcytrus-re.github.io%2F)](https://cytrus-re.github.io) [![Wiki](https://img.shields.io/badge/Wiki-information%20about%20cytrus--re-informational)](https://github.com/Cytrus-RE/cytrus-re/wiki) ![Size](https://img.shields.io/github/repo-size/Cytrus-RE/cytrus-re?label=Cytrus-RE%20Size)
```

Comment:
I don't think we currently have to remove it, maybe just add a small disclaimer about it being broken and not recommended? (we will think about ditching it later if rexo wants to.)

Reply:
If you have suggestions, please don't request changes for them. Also, no, that's not the point of the PR. At the moment, it looks like this:
![image](https://user-images.githubusercontent.com/57069715/113866102-3177c000-97ad-11eb-81ae-a338eedcebef.png)
and before, when you could actually see the svg file, it was inconsistent with the other icons.

Reply:
The actual link works fine so maybe we should just move that somewhere else? TBH I'm fine with removing it since, well, codespaces + it's easy to get to Gitpod

Reply:
we should probably remove it. don't think anyone particularly cared about it anyway

Reply:
LGTM, actually (also yeah sorry i thought we had to change stuff)
try to move it somewhere else actually.